docs/gen-docs/kyma_provision_aks.md

```diff
@@ -0,0 +1,41 @@
+## kyma provision aks
```

Comment:
We probably have to manually delete the old docs file `kyma_provision_azure.md`

Reply:
done